Data-Science/Ray/Ray-CPU/Hyperparameter/README.md

```diff
@@ -0,0 +1,24 @@
+## Ray Hyperparameter Example
+
+##### This example demonstrates hyperparameter tuning in a distributed computing environment using Ray, specifically applied to training a news recommendation system.
+
+### Prerequisites:
+* Ensure you are using a Data Science notebook environment in Kubeflow.
+* Ray client and server versions must match. Typically, `ray --version` can be used to verify the installed version.
+* Activate the Ray-specific Python kernel in your notebook environment.
+* For optimal performance, ensure the working directory contains only the necessary files for the Ray job. `json` data files are located `resources` directory.
```

Comment:
Same for this one too. I think we need to specify the exact working directory and exact files that are necessary to submit this Ray job.